### ISSUE TITLE
[BUG] Icons overlap text in 3.2.0

### ISSUE THREAD

**Issue by rohankapoorcom** [bitfocus/companion#2718]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I upgraded from 3.1.2 to 3.2.0 (3.2.0+6663-stable-b83f412f) today and immediately noticed that all of the icons on buttons are now immediately bigger. For buttons with no text, this is okay, but for buttons that had an icon and the text, the icon now overlaps the text.

Previously I had the text alignment set to lowered centered and the icon alignment set to upper centered and this neatly laid out both without overlap. Now the same buttons have the overlap.

### Steps To Reproduce

1. Create a new button with the text size 18pt with a single word like "Off" and the text alignment set to bottom center.
2. Upload an image sized 48x48 (examples attached)
![fan](https://github.com/bitfocus/companion/assets/2511957/f3a7cdbc-5271-479c-a8ec-d99d05059190) and set the image alignment to upper center.
3. Observe that the text is covered by the icon.

### Expected Behavior

In companion 3.1.2, doing the same results in a button with the text at the bottom and the image at the top with no overlap.

1. Create a new button with the text size 18pt with a single word like "Off" and the text alignment set to bottom center.
2. Upload an image sized 48x48 (examples attached)
![fan](https://github.com/bitfocus/companion/assets/2511957/f3a7cdbc-5271-479c-a8ec-d99d05059190) and set the image alignment to upper center.
3. Observe that the text is not covered by the icon

### Environment (please complete the following information)

```markdown
- OS: Ubuntu 22.04 (Docker Host), Windows 10 / Mac OS X Sonoma (both clients)
- Browser: Chrome 120.0.6099.201
- Companion Version: 3.2.0 (3.2.0+6663-stable-b83f412f)
```

### Additional context

![Capture](https://github.com/bitfocus/companion/assets/2511957/321d17f7-b5f9-42ae-b59c-73aa06d07bb7)
Full page with the overlap is visible here.

**Comment by Julusian** [comment]
this is fixed in v3.2.1

**Comment by rohankapoorcom** [comment]
I can confirm, the issue is gone with v3.2.1! Thanks for the quick fix!
